enhancements/plain-k8s-bundles.md

```diff
@@ -0,0 +1,220 @@
+
+# plain-k8s-bundles
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Add support for a new mediatype, `k8s+v1`, to the [Operator Bundle spec](https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md), [operator-registry](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go), and [OLM](https://github.com/operator-framework/operator-lifecycle-manager) that enables packaging an operator without defining a [ClusterServiceVersion](https://github.com/operator-framework/olm-book).
+
+## Motivation
+
+Operator authors find CSVs difficult to write and maintain. Expressing an OLM-managed operator as a set of related kubernetes objects is more intuitive to users. Supporting plain kubernetes manifests makes OLM more approachable to the wider community, which is relevant in light of the Operator Framework attempting to become a CNCF project. This feature is part of the overall goal of making OLM simpler to use.
+
+Furthermore, this feature is part of the overall goal of making OLM more flexible by supporting more media types. The more media types OLM supports the more popular and accesible OLM becomes.
+
+### Goals
+
+- Support, install, build, and lifecycle `k8s+v1` bundles  
+- Support adding `k8s+v1` bundles to an [Operator Index](https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-registry.md#summary)
+- Support packageserver surfacing data on plain manifest bundles
+
+### Non-Goals
+
+- Supporting helm bundles
+- Building a bundle without providing any additional metadata in some way
+- Implement operatorhub.io usage of this new mediatype
+
+## Proposal
+
+### New Mediatype
+
+The `k8s+v1` mediatype will be used to represent a bundle composed of plain Kubernetes manifests. This will let OLM and other tooling differentiate this type of bundle from other types.
+
+### Inferring CSV Content
+
+CSVs are generally ⅔s standard k8s manifests and ⅓ operator runtime information. They provide a description that can be used to gather dependencies for, install, and display an operator. When attempting to denormalize this description, it's critical that there is no loss of information capacity. To this end, it helps to split it into two categories: inferrable and non-inferrable.
+
+#### Inferrable
+
+To support a wider range of inferred CSV fields, manifests for resources usually generated by a CSV will need to be provided as input to `k8s+v1` bundle creation:
+
+- Deployments for InstallStrategy
+- RBAC w/ subjects matching Deployment ServiceAccounts for Permissions and Owned, Required, and NativeAPIs
+- APIServices and Services for APIServiceDefinitions
```

Comment:
This should also list `ValidatingWebhookConfiguration` and `MutatingWebhookConfiguration`

Reply:
They are listed further down in the implementation details section.